code/modules/cargo/supplypod.dm

```diff
@@ -80,8 +81,35 @@
 	landingDelay = 20 //Very speedy!
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
-/obj/structure/closet/supplypod/Initialize()
+/obj/structure/closet/supplypod/securitypod
+	style = STYLE_SECURITY
+	bluespace = TRUE
+	explosionSize = list(0,0,0,1)
+	var/obj/item/radio/Radio
+
+/obj/structure/closet/supplypod/securitypod/Initialize()
+	. = ..()
+	Radio = new /obj/item/radio(src)
+	Radio.listening = 0
+
+/obj/structure/closet/supplypod/securitypod/Destroy()
+	QDEL_NULL(Radio)
+	return ..()
+
+/obj/structure/closet/supplypod/securitypod/preOpen()
+		Radio.set_frequency(FREQ_SECURITY)
+		Radio.talk_into(src, "SECURITY ALERT: New prisoner dropping into [get_area(src)].", FREQ_SECURITY)
```

Comment:
can you double check that set_frequency is necessary? i suspect that's for continued transmissions or listening, you probably only need to do the talk_into based on the fact it takes a frequency but im just guessing

Reply:
it is